tests/functions/module_web.test.ts

```diff
@@ -0,0 +1,28 @@
+import { evaluateCellText } from "../test_helpers/helpers";
+
+describe("HYPERLINK formula", () => {
+  test("The evaluated result is the link", () => {
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK("https://www.odoo.com", "Odoo")' })).toBe(
+      "Odoo"
+    );
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK("https://www.odoo.com")' })).toBe(
+      "https://www.odoo.com"
+    );
+    expect(evaluateCellText("A1", { A1: "=HYPERLINK(A2)", A2: "https://www.odoo.com" })).toBe(
+      "https://www.odoo.com"
+    );
+    expect(
+      evaluateCellText("A1", { A1: '=HYPERLINK(A2, "Odoo")', A2: "https://www.odoo.com" })
+    ).toBe("Odoo");
+    expect(
+      evaluateCellText("A1", { A1: "=HYPERLINK(A2, A3)", A2: "https://www.odoo.com", A3: "Odoo" })
+    ).toBe("Odoo");
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK("invalidUrl")' })).toBe("invalidUrl");
+    expect(evaluateCellText("A1", { A1: '=HYPERLINK("")' })).toBe("[]()");
```

Comment:
that's weird. Does that mean it will display an empty link component ?
It should probably not be a link and just left empty.
It's a good test to add in `cell.test.ts`: check that `cell.link` is undefined in those situations
same for the line below